packages/main/src/Calendar.js

```diff
@@ -416,11 +468,83 @@ class Calendar extends UI5Element {
 				this._hideMonthPicker();
 			}
 		}
+
+		if (isTabNext(event)) {
+			const header = this.shadowRoot.querySelector("ui5-calendar-header");
```

Comment:
Create getters for those as well

get header
this.shadowRoot.querySelector("ui5-calendar-header");

get monthBtn
 header.shadowRoot.querySelector("[data-sap-show-picker='Month']");

get yearBtn
header.shadowRoot.querySelector("[data-sap-show-picker='Year']");

Reply:
Done